PkSolver/IO/CSVReader.cxx

```diff
@@ -0,0 +1,54 @@
+#include "CSVReader.h"
```

Comment:
Consider using something off-the-shelf instead? Perhaps https://github.com/ben-strasser/fast-cpp-csv-parser?

Reply:
This one looks indeed nice for integration since it just requires including one header file (which eradicated my concern of adding a dependency for a very small functionality). I will put it on my list to review in a next step.

Reply:
But probably after C++11 becomes a requirement for Slicer (maybe it already is?)

Reply:
I remember they said something about this at the ProjectWeek, but it might be planed for 5.0 only :/